light/store/db/db.go

```diff
@@ -14,10 +13,6 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-var (
-	sizeKey = []byte("size")
```

Comment:
Just a note: I replaced this with `s.sizeKey()` because the light store is supposed to support multiple db's with different prefixes, however if this is the case, then the size key will keep being overwritten by every operation of each of the different db's.